dietpi/dietpi-software

```diff
@@ -7748,16 +7745,15 @@ url.redirect = (
 				fi
 
 				# Cal/CardDAV redirects to ownCloud DAV endpoint
-				if [[ ! -f /etc/nginx/sites-dietpi/dietpi-dav_redirect.config ]]; then
+				if [[ ! -f /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf ]]; then
 
-					#???Enable rewrite mod
 					echo '# Redirect Cal/CardDAV requests to ownCloud endpoint:
 location = /.well-known/carddav {
 	return 301 $scheme://$host/owncloud/remote.php/dav;
 }
 location = /.well-known/caldav {
 	return 301 $scheme://$host/owncloud/remote.php/dav;
-}' > /etc/nginx/sites-dietpi/dietpi-dav_redirect.config
+}' > /etc/nginx/sites-dietpi/dietpi-dav_redirect.conf
```

Comment:
Needs verifying if Nginx supports the ```.conf``` extension. Might require ```.config```?

Reply:
@Fourdee
Jep file ending can be freely chosen by include directive. Also inside conf.d it is .conf, similar to all other webservers. So it's more constant to stay with this.